src/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -7,8 +7,9 @@
     <OmitResources Condition="'$(IsPartialFacadeAssembly)' == 'true'">true</OmitResources>
     <UsePackageTargetRuntimeDefaults Condition="'$(IsPartialFacadeAssembly)' != 'true'">true</UsePackageTargetRuntimeDefaults>
     <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netstandard2.1'">$(DefineConstants);netstandard2_1</DefineConstants>
```

Comment:
Can you please define this constant in `Directory.Build.props` as we do with others and don't use `_`? Just define it as `netstandard21`.